ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -124,7 +124,7 @@ function upgrade() {
     set -x
     TARGET_RELEASES="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
     if [[ -f "${SHARED_DIR}/override-upgrade" ]]; then
-        TARGET_RELEASES="$(< "${SHARED_DIR}/override-releases")"
+        TARGET_RELEASES="$(< "${SHARED_DIR}/override-upgrade")"
```

Comment:
is this in purpose?

Reply:
Fixes a typo in previous PR - https://github.com/openshift/release/pull/17823